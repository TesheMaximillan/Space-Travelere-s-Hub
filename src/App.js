import React from 'react';
import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
import './App.scss';
import Myprofile from './components/common/Myprofile';
import Navbar from './components/common/Navbar';
import Missions from './routes/Missions';
import Rockets from './routes/Rockets';

function App() {
  return (
    <Router>
      <Navbar />
      <Routes>
        <Route path="/" element={<Rockets />} />
        <Route path="/missions" element={<Missions />} />
        <Route path="/myprofiles" element={<Myprofile />} />
      </Routes>
    </Router>
  );
}

export default App;
