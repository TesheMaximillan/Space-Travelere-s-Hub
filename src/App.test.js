import renderer from 'react-test-renderer';
import renderContext from './components/storeMock';

import App from './App';

describe('Myprofile component', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(renderContext(<App />));
  });

  it('should render with given state from redux store', () => {
    // Assert
    expect(component.toJSON()).toMatchSnapshot();
  });
});
