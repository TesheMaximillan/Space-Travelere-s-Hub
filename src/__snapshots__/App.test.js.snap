// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Myprofile component should render with given state from redux store 1`] = `
Array [
  <div
    className="nav"
  >
    <div
      className="logoContainer"
    >
      <img
        alt="Logo"
        className="logoImg"
        src="logo192.png"
      />
      <h1
        className="logoText"
      >
        Space Traveler's Hub
      </h1>
    </div>
    <ul
      className="navItems"
    >
      <li
        className="item"
      >
        <a
          aria-current="page"
          className="itemLink selectedItem"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li
        className="item"
      >
        <a
          className="itemLink"
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li
        className="item"
      >
        <a
          className="itemLink"
          href="/myprofiles"
          onClick={[Function]}
        >
          Myprofile
        </a>
      </li>
    </ul>
  </div>,
  <div
    className="rocketsContainer"
  />,
]
`;
